Gemfile

```diff
@@ -59,6 +59,10 @@ group :development do
   gem "activerecord-deprecated_finders"
 end
 
+group :development, :test do
+  gem 'pry'
```

Comment:
Oh, right. Sorry. This is the only project I've worked on that actually uses Spring.